README.adoc

```diff
@@ -230,16 +232,26 @@ In this case, the `setup()` method creates a JAX-RS client that makes HTTP reque
 inventory service. Register this client with a `JsrJsonpProvider` JSON-P provider
 to process JSON resources. The `teardown()` method destroys this client instance.
 
-The following list describes the test cases that you can run:
-
-* Use the `testGetPropertiesTime()` test case to test the `@Timed` annotation. This test case sends a request to the `http://localhost:9080/inventory/systems/localhost` URL. The test verifies the `200` response code, which is returned with the `connectToEndpoint()` private method. Next, the test case makes a connection to the `https://localhost:9443/metrics/application` URL to get the content of the
-metrics as plain text. The test case then uses the `validateMetric()` method to assert whether the time that is needed to get the system properties for `localhost` is less than 4 seconds.
-
-* Use the `testListCount()` test case to test the `@Counted` annotation. This test case uses the `connectToEndpoint()` method to send a request to the `http://localhost:9080/inventory/systems` URL. Next, the test case uses the `validateMetric()` method to assert whether the `list()` method in the `InventoryManager` class was invoked once.
-
-* Use the `testInventorySize()` test case to test the `@Gauge` annotation. This test case uses the `connectToEndpoint()` method to send a request to the `http://localhost:9080/inventory/systems/localhost` URL. The inventory grows in size because of the request. The
-`getHostCount()` method in the `InventoryManager` class returns one host, which represents the
-localhost. The request then asserts the host with the `validateMetric()` method.
+These are the break down of the test cases:
```

Comment:
The following list shows explanations of the test cases:

Reply:
I like "Here are the break down of the test cases:" @sishida

Reply:
=> "See the break down of the test cases:"

Reply:
Fixed

Reply:
@evelinec and @Ahmad-Ayyoub: Sorry, I just realized..."breakdown" should be one word, not two.

Reply:
Update: I fixed it and merged directly to master.